internal/display/roles.go

```diff
@@ -0,0 +1,78 @@
+package display
+
+import (
+	"github.com/auth0/auth0-cli/internal/ansi"
+	"gopkg.in/auth0.v5/management"
+)
+
+type roleView struct {
+	Name        string
+	ID          string
+	Description string
+}
+
+func (v *roleView) AsTableHeader() []string {
+	return []string{"Name", "Role ID", "Description"}
+}
+
+func (v *roleView) AsTableRow() []string {
+	return []string{
+		v.Name,
+		ansi.Faint(v.ID),
```

Comment:
Can we have the ID go first? Like in the other list commands.

Reply:
Yep, done